dom.bs

```diff
@@ -1217,7 +1217,32 @@ for discussion).
    <a>invoke</a> algorithm.
 
    <li><p>If <var>event</var>'s {{Event/type}} attribute value is not <var>listener</var>'s
-   <b>type</b>, terminate these substeps (and run them for the next <a>event listener</a>).
+   <b>type</b>, check for a legacy type attribute match by running these substeps:
+
+   <ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationend":
+    <ol>
+     <li><p>Let <var>original event type</var> be "animationend".
```

Comment:
Introducing a variable in substeps seems suspect with respect to scoping. Best to introduce it earlier and just set it here.